src/raftstore/store/peer.rs

```diff
@@ -1318,6 +1318,11 @@ impl Peer {
                 if lease_to_be_updated {
                     let propose_time = self.find_propose_time(entry.get_index(), entry.get_term());
                     if let Some(propose_time) = propose_time {
+                        ctx.raft_metrics.commit_log.observe(duration_to_sec(
+                            (ctx.lease_time.unwrap_or_else(monotonic_raw_now) - propose_time)
```

Comment:
why use lease time here?

Reply:
you can see #5421 for the reason to use lease time